autoload/tagbar.vim

```diff
@@ -1906,13 +1906,13 @@ function! s:RenderContent(...) abort
     if !empty(tagbar#state#get_current_file(0)) &&
      \ fileinfo.fpath ==# tagbar#state#get_current_file(0).fpath
         let scrolloff_save = &scrolloff
-        set scrolloff=0
+        setlocal scrolloff=0
 
         call cursor(topline, 1)
         normal! zt
         call cursor(saveline, savecol)
 
-        let &scrolloff = scrolloff_save
+        execute 'setlocal scrolloff=' . scrolloff_save
```

Comment:
```suggestion
        let &l:scrolloff = scrolloff_save
```
I see why execute gets the job don, but I'm not a fan of composing commands as strings and passing them to be evaled when there is a direct method available.